projects/contacts-admin/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.7'
+
+x-contacts-admin: &contacts-admin
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: contacts-admin
+  stdin_open: true
+  tty: true
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/contacts-admin
+
+services:
+  contacts-admin-lite:
+    <<: *contacts-admin
+    privileged: true
+    depends_on:
+      - mysql
+      - redis
+    environment:
+      DATABASE_URL: "mysql2://root:root@mysql/contacts_development"
+      TEST_DATABASE_URL: "mysql2://root:root@mysql/contacts_test"
+      REDIS_HOST: redis
+
+  contacts-admin-app:
+    <<: *contacts-admin
+    depends_on:
+      - mysql
```

Comment:
I think we're missing some dependencies here, in comparison to the Makefile. For example, there's no point `make`ing Whitehall if we don't fire it up anywhere. If it's too inconvenient to test the `-app` stack, we don't have to add it here - someone else can do that when they need to. We could make a note of that in the compatibility list.

Reply:
I'm just going by what I know is still relevant from [the contacts-admin README](https://github.com/alphagov/contacts-admin#dependencies). I last touched this app in 2016. 😬 The thing I was trying to do on Friday is done now anyway thanks to this working `-lite` stack: https://github.com/alphagov/contacts-admin/pull/731. 😅